source/beamline/AMPVControl.h

```diff
@@ -114,6 +114,12 @@ class AMReadOnlyPVControl : public AMControl {
 	///  This extra signal is specialized to report on PV channel connection status.  You should be free to ignore it and use the signals defined in AMControl.
 	void readConnectionTimeoutOccurred();
 
+public slots:
+	/// Sets the minimum value.
+	void setMinimumValue(double newValue);
+	/// Sets the maximum value.
+	void setMaximumValue(double newValue);
```

Comment:
I do wonder if these functions should be given a name which more reflects what is going on here. Essentially these functions are used for acquaman to manually take control of the limits. Once these have been called acquaman will now ignore the limits which come in from epics land. 

Maybe
```C++
void setMinimumValueOverride(double newValue);
void setMaximumValueOverride(double newValue);
```
Or something of that nature???